.github/workflows/demos_notebook_tests.yml

```diff
@@ -111,11 +115,13 @@ jobs:
         host: ${{ secrets.TEST_SYSTEM_IP }}
         username: ${{ secrets.TEST_SYSTEM_USERNAME }}
         key: ${{ secrets.TEST_SYSTEM_MLRUN_PEM }}
+        command_timeout: 20m
         script: |
           jupyter_pod=$(kubectl -n mlrun get pods -o name | grep jupyter | head -n 1 | cut -d / -f 2)
           kubectl exec -n mlrun $jupyter_pod -- /bin/sh -c "jupyter nbconvert --to notebook --execute /home/jovyan/demos/mask-detection/3-automatic-pipeline.ipynb --ExecutePreprocessor.kernel_name=python3 --output=/home/jovyan/demos/mask-detection/3-automatic-pipeline.ipynb"
 
     - name: Deleting junk
+      if: ${{ always() }}
```

Comment:
this seems redundant

Reply:
when a step fails `deleting junk` wasn't executing leaving junk.
adding that will make `deleting junk` run regardless of failing steps